src/main/java/com/volasoftware/tinder/model/Verification.java

```diff
@@ -0,0 +1,73 @@
+package com.volasoftware.tinder.model;
+
+import jakarta.persistence.*;
+
+import java.time.LocalDateTime;
+
+import static jakarta.persistence.TemporalType.TIMESTAMP;
+
+@Entity
+@Table(name = "token")
+public class Verification {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "ID")
+    private Long id;
+
+    @Column(name = "TOKEN")
+    private String token;
+    @Temporal(TIMESTAMP)
+    @Column(name = "CREATED_DATE")
+    private LocalDateTime createdDate;
+    @Temporal(TIMESTAMP)
+    @Column(name = "EXPIRATION_DATE")
+    private LocalDateTime expirationDate;
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "USER_ID")
+    private User userId;
+
+    public User getId() {
```

Comment:
the field, getter, and setter are misleading you are not getting the user ID only, but the whole user, try to make it more clear